compiler/src/iree/compiler/Codegen/Utils/GPUUtils.cpp

```diff
@@ -35,6 +35,7 @@
 #define DBGSNL() (llvm::dbgs() << "\n")
 
 constexpr unsigned kShuffleBitWidth = 32;
+constexpr char kDenormalFpMathF32AttrName[] = "denormal_fp_math_f32";
```

Comment:
nit: I think it's useful to namespace floating attributes like this since it makes it clearer who the intended consumers are and reduces the probability of conflicts (though the chance of a conflict here is admittedly unlikely).
```suggestion
constexpr char kDenormalFpMathF32AttrName[] = "iree_codegen.denormal_fp_math_f32";
```